package.json

```diff
@@ -35,7 +35,7 @@
     "buffer": "6.0.3",
     "html-webpack-plugin": "5.5.0",
     "webpack": "5.67.0",
-    "webpack-cli": "4.9.2",
+    "webpack-cli": "4.10",
```

Comment:
Let's make a new `plutip-test` entry in `package.json` (we should also run it as part of `npm run test`)